src/Editor/GUI/FileSelector.hpp

```diff
@@ -19,11 +19,17 @@ namespace GUI {
              */
             void SetFileSelectedCallback(std::function<void(const std::string&)> callback);
             
-            /// Set extension to select files of.
+            /// Set extensions to select files of.
             /**
-             * @param extension Set the extension of which files to show.
+             * @param extensions Set the extensions of which files to show.
```

Comment:
Should be a description of the parameter, not what the function does.

Remove "Set the".